Gemfile

```diff
@@ -4,6 +4,15 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem 'rack'
+gem 'rake'
+%w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |repo|
```

Comment:
Could we put this into a Gemfile.local on CI? Like how we test against RuboCop master branch.

Reply:
Done 👍